terraform/horcrux/variables.tf

```diff
@@ -36,90 +36,90 @@ variable "num_sentry_nodes" {
 # Machine Types
 variable "horcrux_machine_type" {
   description = "The machine type for the horcrux nodes"
-  default = "n1-highcpu-2"
+  default     = "n1-highcpu-2"
 }
 
 variable "sentry_machine_type" {
   description = "The machine type for the sentry nodes"
-  default = "n2d-standard-4"
+  default     = "n2d-standard-4"
 }
 
 # Image Types
 variable "horcrux_image_type" {
   description = "The image type for the horcrux nodes"
-  default = "COS"
+  default     = "COS"
```

Comment:
We should update this one and the one right below it to `COS_CONTAINERD` while we're here